cli/azd/cmd/infra.go

```diff
@@ -32,6 +32,8 @@ func infraActions(root *actions.ActionDescriptor) *actions.ActionDescriptor {
 			Command:        newInfraDeleteCmd(),
 			FlagsResolver:  newInfraDeleteFlags,
 			ActionResolver: newInfraDeleteAction,
+			OutputFormats:  []output.Format{output.JsonFormat, output.NoneFormat},
+			DefaultFormat:  output.NoneFormat,
```

Comment:
Looking at the history, I think this was always the case, where we missed this option on the aliased command. I think it speaks to a possible design improvement where `flags` should control the output format, and aliases would share flags and eliminate this type of problem. 

Do you think this type of change is doable? I've been meaning to make some changes around this.

Reply:
Agree on the flags change, but going to hold off on this PR.  I have a POC, but it ended up being larger than I would have liked for this current PR.